dkam/pkg/script/Installer

```diff
@@ -347,6 +348,7 @@ install_hardware_discovery_agent(){
 		echo "Skipping install_hardware_discovery_agent"
     else
         echo "Install Hardware discovery agent..."
+        apt-get update
```

Comment:
Do we need to do `apt-get update` for every agent installation?

Reply:
Should be benign. Will help in cases where there's a race whether in the local env setup or the package hosting itself.